### ISSUE TITLE
<help content> Update SC 4.1.1 requirements and references in the rule help 

### ISSUE THREAD

**Issue by shunguoy** [IBMa/equal-access#1681]
Dependent on #1680

currently, 5 rules are mapped to 4.1.1 in the rulesets mapping. Each of the mappings needs to be updated to a different SC. Corresponding test cases or baselines may need to be updated accordingly.

Help files will need to change accordingly

**Comment by shunguoy** [comment]
branch: dev-1680.  @philljenkins

**Comment by philljenkins** [comment]
Changes made to the 7 help files
1. element_id_unique, 
2. element_accesskey_unique, 
3. element_accesskey_labelled, (not a mapping to 4.1.1)
4. element_attribute_deprecated, 
5. aria_attribute_deprecated, 
6. aria_role_redundant

@shunguoy Please paste a screenshot of the 6 new Help file here for review

**Comment by philljenkins** [comment]
Simplifying the Note to remove examples not related to the specific Help file in which it is placed

> Specification issues such as missing roles due to inappropriately nested elements, deprecated roles, incorrect states or names due to a duplicate ID are covered by other accessibility requirements and will be reported under those requirements rather than the previous "4.1.1 Parsing" requirement that was removed in WCAG 2.2.

**Comment by philljenkins** [comment]
@tombrunet It's ready for your review as part of PR #1686

**Comment by philljenkins** [comment]
Tom said: "Maybe it's because there's no punctuation, but I can't parse that sentence."
>Note: Only when specification issues such as using deprecated attributes that also cause accessibility issues that are checked by other rules will be reported under those requirements rather than the previous "4.1.1 Parsing" requirement that was removed in WCAG 2.2.

Phill replied: "I had a similar problem, but running it through 2 Grammar checkers didn't flag an issue.  I think I will split it into two sentences."

**Comment by philljenkins** [comment]
Since we're adding new category of rules, "**_HTML_**" and "**ARIA**", the user will see automatically see new information displayed on the [Checker rule sets](https://www.ibm.com/able/requirements/checker-rule-sets) page.

- [ ] add a descriptive paragraph for these two new categories. I'll add the errata paragraph from the W3C that is removing 4.1.1 Parsing

See [...accessibility-checker-engine/src/v4/rulesets.ts#L26](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rulesets.ts)

**Comment by philljenkins** [comment]
@tombrunet @shunguoy Proposed new note separated into two sentences for readability:

Note: The previous [4.1.1 Parsing](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html) requirement was removed from WCAG. However, when specification issues also cause accessibility issues they will be reported against the other accessibility requirements when they are detected by the other rules.

**Comment by philljenkins** [comment]
@tombrunet @shunguoy  please review the new Proposed categories that will be automatically displayed in the [Checker rule set ](https://www.ibm.com/able/requirements/checker-rule-sets) descriptions:

~~These rules were originally adopted to address problems that assistive technology had directly parsing HTML. Since these rules were written~~ 

`"HTML": "The HTML Standard has adopted specific requirements governing how user agents must handle issues such as incomplete tags, incorrect nesting, deprecated elements, duplicate attributes, and incorrect states or names due to a duplicate ID. Now specification issues that cause accessibility issues may be covered by different rules and will be reported under those accessibility requirements rather than as issues with "4.1.1 Parsing". However, non-conforming HTML specification issues are still reported.",`

~~These rules were originally adopted to address problems that assistive technology had directly parsing ARIA. Since these rules were written,~~

`"ARIA": "The ARIA specifications have adopted requirements governing how user agents must handle issues such as incomplete tags, deprecated or redundant roles, incorrect nesting, duplicate attributes, or incorrect states or names due to a duplicate ID. Specification issues that cause accessibility issues may be covered by different rules and will be reported under those accessibility requirements rather than as issues with "4.1.1 Parsing". However, non-conforming ARIA specification issues are still reported.",`

**Comment by philljenkins** [comment]
Here is the as-is [Checker rule set](https://www.ibm.com/able/requirements/checker-rule-sets/) page.  

"HTML" and "ARIA" will be bold like the "1.1.1 Non-text..." heading and then the description sentence above will follow the heading. Then the individual rules that map to "HTML" will then follow with links to their Help page ...
![image](https://github.com/IBMa/equal-access/assets/16193609/ad5d12f9-db2b-44e1-907e-18e97c1946f3)

 
image.png

**Comment by philljenkins** [comment]
@tombrunet Help files and ruleset file updated and ready for your re-review as part of PR https://github.com/IBMa/equal-access/pull/1686

Is there a way to test the "auto" update to the [Checker rule set](https://www.ibm.com/able/requirements/checker-rule-sets/) page without deploying?
